src/logic/operate.js

```diff
@@ -0,0 +1,17 @@
+import Big from 'big.js';
+
+const operate = (numOne, numTwo, operation) => {
+  const x = Big(numOne);
+  const y = Big(numTwo);
+  let result = 0;
+
+  if (operation === '+') return x.plus(y);
```

Comment:
You need to add the last condition, `if operation === %`

![image](https://user-images.githubusercontent.com/31897434/108831165-0a34ad00-75ca-11eb-8a54-b9b2cc4a21e2.png)

[checkout Pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/)